spring-cas/pom.xml

```diff
@@ -47,6 +47,7 @@
 		<dependency>
 			<groupId>org.springframework.security</groupId>
 			<artifactId>spring-security-oauth2-resource-server</artifactId>
+			<scope>provided</scope>
```

Comment:
I tested this with both samples (spring-security-cas and the sample with xsuaa) and it is working!